src/containers/Detail/index.tsx

```diff
@@ -0,0 +1,67 @@
+// @flow
```

Comment:
What is it?

Reply:
when i use extestion vscode to create function component,that it apeard, i'm going to remove this.